### ISSUE TITLE
Add term flat thorax

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10370]
**Preferred term label:**
Flat thorax

**Synonyms**
flat chest

**Definition (free text, please give PubMed ID)**
Reduced anteroposterior diameter of the thorax

**Parent term (use hpo.jax.org/app)**
Abnormal thorax morphology HP:0000765

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
pleuroparenchymal fibroelastosis (PMID: 24881083)
3M-syndrome (PMID33258289)

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
this was
HP:6000327
Thoracic cage flatness

**Comment by azankl** [comment]
@pnrobinson Agree, sorry we missed that. But can we then add `flat thorax` as a synonym to `HP:6000327`. I also prefer the definition `Reduced anteroposterior diameter of the thorax` to the current definition of `HP:6000327`.

And can we please add this term as an association to 3M Syndrome 1 (CUL7-associcated, OMIM 273750) based on patient 4 in [PMID:33258289](https://pubmed.ncbi.nlm.nih.gov/33258289/), who has a CUL7 mutation and this feature?
